walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/TimePoint.java

```diff
@@ -0,0 +1,61 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+import java.util.Comparator;
+
+/** Class representing a point of time */
+public final class TimePoint {
+  public static final int START_OF_DAY = getTimeInMinutes(0, 0);
```

Comment:
`getTimeInMinutes` is only used here. I suggest simply writing `END_OF_DAY = 60 * 24 + 59;` and deleting the function. This would be clear enough to anyone reading the code.